src/battle-animations.ts

```diff
@@ -104,6 +104,17 @@ class BattleScene {
 			return '???pokemon:' + pokemonId + '???';
 		};
 
+		if (window.soundManager) {
```

Comment:
I hope this has been tested? It seems wayyyyy too late. It should nearly definitely be at the top level.

Reply:
I re-tested just now and both music and cries seem to work just fine in battles (and replays!). Happy to move it if you would like though.

Reply:
Yeah, even if it works, I'd rather have it top-level (so it would be preloaded, instead of needing to wait until the battle starts to star the load, which could take some time on older systems).

Reply:
Sure, good point.